www/userguide/userguide.md

```diff
@@ -0,0 +1,126 @@
+\page userguide User Guide
+\tableofcontents
+
+**CodeChecker** is a static analysis infrastructure built on the [LLVM/Clang
+Static Analyzer](http://clang-analyzer.llvm.org) toolchain, replacing
```

Comment:
Is CodeChecker really going to replace scan-build, or is it an alternative tool with much more functionalities? And if it really replaces scan-build, will the user guide reader know what scan-build was originally?